src/rred_reports/reports/schools.py

```diff
@@ -108,10 +108,12 @@ def filter_by_entry_and_exit(school_dataframe: pd.DataFrame, report_year: int) -
     """
     report_start, report_end = trial_period_dates(report_year)
 
-    return school_dataframe.loc[
-        ((school_dataframe["entry_date"] > report_start) & (school_dataframe["entry_date"] < report_end))
-        | ((school_dataframe["exit_date"] > report_start) & (school_dataframe["exit_date"] < report_end))
-    ]
+    return (

```

Comment:
might be worth just adding this to the `school_filter` so we don't have to have it replicated 3 times, what do you think?

Reply:
yes good plan! will do that now

Reply:
ah it fails on a test when this happens so may just keep it as is for now!